src/app/theme/legends-theme.ts

```diff
@@ -0,0 +1,26 @@
+import { Theme } from "./symbols";
+
+export const legendsTheme: Theme = {
+  key: "legends",
+  name: "Legends Theme",
+  properties: {
+    "--background": "#1c1812",
+    "--text": "#EFF3F8",
+    "--grey": "#999",
+    "--secondary": "#2d261c",
+    "--secalt": "#14110e",
+    "--textalt": "#eee",
+    "--norm": "#eee",
+    "--formbg": "#14110e",
+    "--link": "#D3B882",
+    "--hover": "#e5c075",
+    "--border": "#382418",
+    "--mborder": "#382418",
+    "--filter": "invert(98%) sepia(1%) saturate(264%) hue-rotate(181deg) brightness(116%) contrast(100%)",
+    "--unread": "#1a293f",
+    "--topbar": "#16130d",
+    "--cblue": "#238eff",
+    "--cred": "#e0245e",
+    "--cgreen": "#17BF63",
```

Comment:
just added a new property `--loading` which sets the color of the loading spinner. It uses the same value as `--grey` for dark and icy dark, but uses `--textalt` for light theme.

Reply:
I'm going to create a PR to add typing for properties shortly.